indra/newview/llviewermenu.cpp

```diff
@@ -9395,16 +9395,19 @@ void initialize_edit_menu()
 
 }
 
+typedef LLUICtrl::LLCommitCallbackInfo cb_info;
+#define COMMIT_ADD(func_name, func) LLUICtrl::SharedCommitCallbackRegistry::currentRegistrar().add(func_name, cb_info(func))
+#define COMMIT_ADD_TRUSTED(func_name, func) LLUICtrl::SharedCommitCallbackRegistry::currentRegistrar().add(func_name, cb_info(func, cb_info::UNTRUSTED_BLOCK))
```

Comment:
Instead of two macros, how about one (inline?) function with a default `EUntrustedCall` value? That lets you pass `UNTRUSTED_THROTTLE` too.